inspirehep/modules/records/receivers.py

```diff
@@ -206,3 +215,45 @@ def enhance_after_index(sender, json, record, *args, **kwargs):
 
     elif is_data(json):
         populate_citations_count(record=record, json=json)
+
+
+@after_record_insert.connect
+@after_record_update.connect
+def after_update(sender, *args, **kwargs):
```

Comment:
I'd name this function after what it does rather than when it does it.
Or maybe follow the patter: what_when, like `def reindex_referenced_records_after_record_update(...)`

Reply:
done